hanadb_exporter/db_manager.py

```diff
@@ -60,9 +60,13 @@ def _connect_tenants(self, host, connection_data):
             # If userkey is used database name must be added to connect to tenants
             if connection_data.get('userkey'):
                 connection_data['databaseName'] = database
-            conn.connect(
-                host, tenant_port, **connection_data)
-            self._db_connectors.append(conn)
+            try:
+                conn.connect(
+                    host, tenant_port, **connection_data)
+                self._db_connectors.append(conn)
+                self._logger.info('Connected succesfully to TENANT dataase %s ', database )
+            except:
```

Comment:
I don't like this so generic `except`. Could you just catch the error that we really want to create the exception for?

By the way, remove the last space before closing the parenthesis

Reply:
Ok, so this will be an exception raise during the `connect` method. This method has specific exceptions, so catch the ones you need here

Reply:
implemented